docs/Makefile

```diff
@@ -229,20 +240,16 @@ dummy:
 	@echo
 	@echo "Build finished. Dummy builder generates no files."
 
+# this is only left for backwards compatibility. Changed to be consistent with windows
 .PHONY: html-api
-html-api:
+html-api: htmlapi
```

Comment:
This is super confusing to have the target `html-api` that depends on `htmlapi`.

Reply:
well, backwards compatibility :)

Reply:
OK now I get it. Its for compatibillity. good job!